tests/unit/static/golden/api.trulens.3.11.yaml

```diff
@@ -4,7 +4,7 @@ trulens:
   lows: {}
 trulens.benchmark:
   __class__: builtins.module
-  __version__: 1.2.2
+  __version__: 1.2.6
```

Comment:
what are these files for?

Reply:
there's an API test that @sfc-gh-pmardziel put together that reads the spec from this file